blog/2024-05-16-recharging-tranche-strategy-tutorial.md

```diff
@@ -0,0 +1,306 @@
+---
+slug: recharging-tranche-strategy-tutorial
+title: "Deploying a market making strategy with Raindex"
+authors: dcatki
+tags: [market making, strategy, raindex]
+---
+
+Efficient market making is crucial for providing liquidity and stabilizing token prices. Raindex offers a powerful toolset for deploying bespoke market making strategies that can dynamically respond to market conditions.
+
+These strategies are:
+- 100% onchain and perpetual (trade while you sleep)
+- Self custodial (nobody else handles your tokens)
+- Don't require any bot infra (the open Raindex solver network actually clears transactions)
+
+One such strategy is the Recharging Tranche Strategy, which ensures continuous liquidity and effective price discovery through an innovative recharging mechanism.
+
+## What is the Recharging Tranche Strategy?
```

Comment:
i thought we were going to rename everything to "grid"?

Reply:
hmm only thing is I think we'll run into that thing where people expect it to be the same as a grid strategy, not just _similar_ to one.

Reply:
@hardyjosh if you set recharge to 0 is it not the same as a grid strategy?

Reply:
@thedavidmeister i'll rename, we can try it with people